app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::ItemsController < ApplicationController
+  def create
+    if item_params.keys.count == 4
```

Comment:
Does it matter what the four keys are? If so, you probably want to specify those here instead of checking that the count is 4.

Reply:
I was thinking that the evaluation of the keys that happens in the params.permit was enough to go off of. If 4 keys are present, then they must be the correct keys. Does it work that way?

Reply:
I don’t think you’ll need to check that as long as you have model-level validations and are checking if item.save